### ISSUE TITLE
Rcon tool?

### ISSUE THREAD

**Issue by cronner** [FezVrasta/ark-server-tools#851]
Anyone any  good Rcon tool for seeing whos is in tribe, get chat, kick, ban, seeing logs, etc?

**Comment by Provokan** [comment]
For Android, Ios or Windows? 

Use fucking Google.

**Comment by cronner** [comment]
Windows and fucking Google does come up with a result that I can use, pretty bad all the rcon clients i've found, so maybe someone in here knows a good one that they use for their server

**Comment by Provokan** [comment]
http://sparcmx.com/commander/

**Comment by cronner** [comment]
Not a Rcon server tool. it's a tool without rcon support for "cheating" in singleplayer?

**Comment by Provokan** [comment]
its a rcon tool, u isr this on my server

**Comment by klightspeed** [comment]
One tool that was created when RCON was initially implemented in ARK is:
http://forum.uhostpro.com/topic/44/arkon-a-comprehensive-guide-for-rcon-server-management/2

A (non-free) web service designed for advanced server management is:
https://www.battlemetrics.com/rcon/about

If you have BattlEye enabled, you can use the BattlEye client:
https://www.battleye.com/downloads/

Console (and RCON) commands are documented at:
https://ark.gamepedia.com/Console_Commands

Note that RCON commands are typically the same as `admincheat` console commands, except without the `admincheat`, and UI commands (including commands that require a target) will not work.

**Comment by cronner** [comment]
Yeah wanted a complete tool that are free for easy managing, Arkon is pretty bad.

maybe someone can make a guide to make this work :)
https://github.com/knightzac19/NodeJS-ArkService
